src/DetectChanges/BCBreak/PropertyBased/PropertyDocumentedTypeChanged.php

```diff
@@ -34,8 +35,13 @@ public function __invoke(ReflectionProperty $fromProperty, ReflectionProperty $t
             return Changes::empty();
         }
 
-        $fromTypes = array_unique($fromProperty->getDocBlockTypeStrings());
-        $toTypes   = array_unique($toProperty->getDocBlockTypeStrings());
+        try {
+            $fromTypes = array_unique($fromProperty->getDocBlockTypeStrings());
+            $toTypes   = array_unique($toProperty->getDocBlockTypeStrings());
+        } catch (InvalidArgumentException $failedToParseDocblock) {
+            // @TODO #134 improve docblock parsing upstream to remove this generic try-catch
+            return Changes::empty();
```

Comment:
Do we have a way of returning "info" at all? or is it all just BC errors only?

Reply:
I can return something, but then the build would be exiting with `1` here

Reply:
yeah, don't worry then :)